### PR TITLE
WIP: Save credential as QR Code

### PR DESCRIPTION
**Pull request by danlee1025**
Added `save_as_qrcode` function to save credential.

Run `rake "healthcards:vc[payload]"` to generate vc, `credential.png`, QR code containing the credential as payload, will get created.

### CODE REVIEW

File: lib/health_cards/verifiable_credential.rb [hunk 1]
```diff
@@ -44,6 +47,14 @@ def credential
 
     def jwt; end
 
+    def save_as_qrcode
```
Comment (radamson) [inline lib/health_cards/verifiable_credential.rb:50]: Can we add a filepath argument here and then have `QR_CODE_PATH` be the default?

File: lib/health_cards/verifiable_credential.rb [hunk 2]
```diff
@@ -44,6 +47,14 @@ def credential
...
+      qrcode = RQRCode::QRCode.new(credential.to_json, size: 40)
```
Comment (radamson) [inline lib/health_cards/verifiable_credential.rb:51]: I think we need to set `mode: number` too to make it a numeric QR Code.

[Every Health Card can be embedded in a QR Code](https://smarthealth.cards/#numerical-encoding) section of the spec has some details on the QR code settings that we should double check.

Comment (danlee1025) [reply lib/health_cards/verifiable_credential.rb:51]: Thanks, I did not see this QR code section.

Comment (radamson) [reply lib/health_cards/verifiable_credential.rb:51]: It's a pretty new section so I don't think it was actually around when we originally created this task.

**Pull request closed without merging**